src/app/address/[principal]/PageClient.tsx

```diff
@@ -1,17 +1,28 @@
 'use client';
 
-import * as React from 'react';
-
 import { useSuspenseAccountBalance } from '../../../common/queries/useAccountBalance';
 import { useAddressNonces } from '../../../common/queries/useAddressNonces';
 import { hasTokenBalance } from '../../../common/utils/accounts';
 import { AddressTxListTabs } from '../../../features/txs-list/tabs/AddressTxListTabs';
+import { Grid, GridProps } from '../../../ui/Grid';
 import { Stack } from '../../../ui/components';
 import { PageTitle } from '../../_components/PageTitle';
 import { AddressSummary } from './AddressSummary';
 import { StxBalance } from './StxBalance';
 import { TokenBalanceCard } from './TokenBalanceCard';
-import { Wrapper } from './Wrapper';
+
+export function AddressPageLayout(props: GridProps) {
+  return (
+    <Grid
+      gridColumnGap="32px"
+      gridTemplateColumns={['100%', '100%', 'repeat(1, calc(100% - 352px) 320px)']}
+      gridRowGap={['32px', '32px', 'unset']}
```

Comment:
would you mind using Chakra variables for the gap?

Reply:
done